test/config/pretest.js

```diff
@@ -3,7 +3,8 @@
 
 var chaiHttp = require("chai-http");
 const { MongoClient } = require("mongodb");
-const uri = "mongodb://localhost:27017/scicat";
+const testDbName = process.env.MONGODB_TEST || "scicat-test";
+const uri = `mongodb://localhost:27017/${testDbName}`;
```

Comment:
I'm not sure what is the correct solution, but I think that have a separate variable to set the dataset in testing will lead to confusion.
I think we should rely only on the value MONGODB_URI. If you are testing locally, you will change the value of MONGODB_URI according in order to use your local database.

Reply:
I think we should try to avoid relying on MONGODB_URI for testing. 
If you by accident, run the test locally without changing the MONGODB_URI the test will wipe off all the local DB data

Reply:
I see your point, but by using the two environmental variables, we are forcing developers to change the database name in two places instead of one.
Also I feel that we are not minimizing the risk to wipe a local DB data. It is still very much present.

Reply:
I'd suggest to just hardcode `scicat-test-db` for testing